src/commands/copy/copy.ts

```diff
@@ -8,26 +10,34 @@ import { config, registerCli, verbose } from '../common.js';
 const CopyValidator = z.object({ source: z.string(), target: z.string() });
 const CopyManifest = z.array(CopyValidator);
 
+function tryParse(x: string): unknown {
+  if (x.startsWith('[') || x.startsWith('{')) return JSON.parse(x);
+  return JSON.parse(gunzipSync(Buffer.from(x, 'base64url')).toString());
+}
+
 export const commandCopy = command({
   name: 'copy',
   args: {
     config,
     verbose,
+    concurrency: option({ type: number, long: 'concurrency', defaultValue: () => 10 }),
     manifest: restPositionals({ type: string, displayName: 'location', description: 'Manifest of file to copy' }),
```

Comment:
I am not sure to understand what is this input? `manifest` is confusing to me here. Where does it come from? Can you explain?